hw_5_strings_file.py

```diff
@@ -0,0 +1,96 @@
+#1)Из текстового файла удалить все слова, содержащие от трех до пяти символов,
```

Comment:
It is a general comment for all the tasks - the code should be represented with functions

Reply:
Thanks. I did (tried)